pkg/services/ngalert/api/authorization.go

```diff
@@ -163,6 +163,11 @@ func (api *API) authorize(method, path string) web.Handler {
 		http.MethodPost + "/api/v1/ngalert/admin_config",
 		http.MethodGet + "/api/v1/ngalert/alertmanagers":
 		return middleware.ReqOrgAdmin
+
+	// Grafana-only Provisioning Paths
+	case http.MethodGet + "/api/provisioning/policies",
+		http.MethodPost + "/api/provisioning/policies":
+		return middleware.ReqOrgAdmin
```

Comment:
Currently binding provisioning to admin-only.

I think long term, we'd still want some FGAC permissions that are specific to provisioning. It's be _really_ nice to create a "provisioning API key" for your tooling that only has the ability to provision resources. We've discussed this and this likely isn't ideal for the current iteration of FGAC, though. So, admin seems to be the second best alternative.